tests/algorithm/scripts/seed_db.py

```diff
@@ -0,0 +1,43 @@
+import csv
+
+import requests
+from helpers import dict_to_pii
+
+
+def seed_database(csv_file, api_url):
+    MAX_CLUSTERS = 100
```

Comment:
What is the purpose of constraining the number of clusters to 100?

Reply:
The `/seed` api endpoint has a hardcoded limit of 100 clusters to help users to keep requests a reasonable size (around 1 MB). `MAX_CLUSTERS` here just enforces that 100 cluster per batch on the sending side.